pkgs/development/python-modules/pyside/tools.nix

```diff
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ cmake pyside qt4 pysideShiboken ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ pyside qt4 pysideShiboken ];
 
   meta = {
     description = "Tools for pyside, the LGPL-licensed Python bindings for the Qt cross-platform application and UI framework";
```

Comment:
Change `homepage` to `https://wiki.qt.io/PySide`.

Reply:
Ok, done.